src/GraphQL/FieldConfigGenerator/Helper/ImageGallery.php

```diff
@@ -0,0 +1,107 @@
+<?php
+declare(strict_types=1);
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Enterprise License (PEL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+
+namespace Pimcore\Bundle\DataHubBundle\GraphQL\FieldConfigGenerator\Helper;
+
+use GraphQL\Type\Definition\ResolveInfo;
+use Pimcore\Bundle\DataHubBundle\GraphQL\FieldHelper\AssetFieldHelper;
+use Pimcore\Model\Asset;
+use Pimcore\Model\DataObject\AbstractObject;
+use Pimcore\Model\DataObject\ClassDefinition;
+use Pimcore\Model\DataObject\Data\Hotspotimage;
+use Pimcore\Model\Element\AbstractElement;
+use Pimcore\Model\Element\Service;
+
+/**
+ * Class ImageGallery
+ * @package Pimcore\Bundle\DataHubBundle\GraphQL\FieldConfigGenerator\Helper
+ */
+class ImageGallery
+{
+    /**
+     * @var ClassDefinition\Data\ImageGallery
+     */
+    public $fieldDefinition;
+
+    /**
+     * @var ClassDefinition
+     */
+    public $class;
+
+    /**
+     * @var AssetFieldHelper
+     */
+    protected $fieldHelper;
+
+    /**
+     * Objects constructor.
+     *
+     * @param $fieldDefinition
+     * @param ClassDefinition $class
+     */
+    public function __construct(ClassDefinition\Data $fieldDefinition, ClassDefinition $class)
+    {
+        $this->fieldDefinition = $fieldDefinition;
+        $this->class = $class;
+        $this->fieldHelper = \Pimcore::getContainer()->get('pimcore.datahub.graphql.fieldhelper.asset');
```

Comment:
we should already keep https://github.com/pimcore/data-hub/issues/4 in mind and avoid/replace it wherever easily possible